### ISSUE TITLE
[Bug Report] Nowplaying Searchbar-animation

### ISSUE THREAD

**Issue by LogischerJonasHD** [jagrosh/MusicBot#330]
**Since version 0.2.6 the searchbar wich would normaly will update every few seconds will just be stuck in place and even won't update the current song!**

**To Reproduce**
1. Run 'JMusicBot-0.2.6-Windows.jar' (I don't know if the other Version ('JMusicBot-0.2.6.jar', 'JMusicBot-0.2.6-Linux.jar', 'JMusicBot-0.2.6-Mac.jar') are doing the same)
2. Do a command 'nowplaying' short 'np'
3. Now the Searchbar won't update anymore

**Expected behavior**
Normaly this box will update every few seconds but since this version it won't.

**Screenshots**
![image](https://user-images.githubusercontent.com/58107194/69493334-0b9a9580-0eae-11ea-8f2d-822116214548.png)

**Comment by jagrosh** [comment]
Please include a copy of the output from the `debug` command. My guess is that you have `npimages` set to `true` in the config; it needs to be set to `false` to have the updating nowplaying bar.

**Comment by LogischerJonasHD** [comment]
Writing on my smartphone right now so I can't copy the output from the debug command. Npimages is set to true but why did this work before version 0.2.6?

**Comment by jagrosh** [comment]
From the example config:
```
// If you set this, the "nowplaying" command will show youtube thumbnails
// Note: If you set this to true, the nowplaying boxes will NOT refresh
// This is because refreshing the boxes causes the image to be reloaded
// every time it refreshes.

npimages = false
```
This was a bug that was fixed in version 0.2.6
